spec/api-browser-window-spec.js

```diff
@@ -1145,8 +1145,11 @@ describe('BrowserWindow module', function () {
         const initialWebContents = webContents.getAllWebContents()
         ipcRenderer.send('prevent-next-new-window', w.webContents.id)
         w.webContents.once('new-window', () => {
-          assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
-          done()
+          // We need to give it some time so the windows get properly disposed (at least on OSX).
+          setTimeout(() => {
+            assert.deepEqual(webContents.getAllWebContents(), initialWebContents)
+            done()
+          }, 100)
```

Comment:
I did not test the CI result for the master branch directly.
Basically what I did was rerun the offending test locally twice: once with the last public release of electron and then once again with the "fixed" electron version. In both cases the assert was failing (on mac).